### PR TITLE
wip

### PR DESCRIPTION
**Pull request by drapisarda**
Sidebar on Case details
User settings page
Removed margin from footer

### CODE REVIEW

**Comment by drapisarda** [comment]
I realize now that we may not need the Reporter user settings edit page

**Comment by volcanioo** [comment]
> I realize now that we may not need the Reporter user settings edit page

Let's talk about this today man! :) I am excited!

File: src/views/auth/Sidebar.vue [hunk 1]
```diff
@@ -9,16 +9,18 @@
...
-        :to="{ name: 'Dashboard' }"
-      >
-        Dashboard
-      </router-link>
-      <!-- <router-link
-        :to="{ name: 'CompanySettings', params: { id: userUid } }"
-      >
-        Settings
-      </router-link> -->
+      <slot>
```
Comment (volcanioo) [inline src/views/auth/Sidebar.vue:12]: what is the reason for using `slot` here?

File: src/router/index.js [hunk 2]
```diff
@@ -68,11 +68,11 @@ const routes = [
...
   {
-    path: '/company-settings',
-    name: 'CompanySettings',
-    component: CompanySettings,
+    path: '/user-settings',
+    name: 'UserSettings',
+    component: UserSettings,
     meta: {
-      auth: true
+      auth: false
```
Comment (volcanioo) [inline src/router/index.js]: i will make `auth` is `true`, if its okay for you?

**Review by volcanioo — APPROVED** [review]
thannk you man!

File: src/views/auth/Sidebar.vue (hunk 1 above)
Comment (drapisarda) [reply src/views/auth/Sidebar.vue:12]: So we can customize it if needed

File: src/router/index.js (hunk 2 above)
Comment (drapisarda) [reply src/router/index.js]: yes go for it

**Pull request merged**